cro-clib/chain.h

```diff
@@ -437,3 +437,15 @@ CroResult cro_withdraw(const char *tenermint_url_string,
                        int32_t viewkey_count,
                        uint8_t *output,
                        uint32_t *output_length);
+
+/**
+ * # Safety
+ *
+ * Should not be called with null pointers
+ */
```

Comment:
add some example for storage, tendermint url

Reply:
Done, added a c example in the doc.